src/components/Tooltip/Layout.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {ReactNode} from 'react';
+import {block} from '../utils/cn';
+import type {DOMProps, QAProps} from '../types';
+import {HotKey, HotKeyProps} from '../HotKey';
+
+import './Layout.scss';
+
+const b = block('tooltip-layout');
+
+export interface TooltipLayoutProps extends DOMProps, QAProps {
+    title: string;
+    hotkey?: HotKeyProps['value'];
+    description?: ReactNode;
+}
+
+export const TooltipLayout: React.FC<TooltipLayoutProps> = function TooltipLayout(props) {
```

Comment:
Imho, naming `TooltipLayout` tells nothing about purpose of component. It should be `ActionControlTooltip` maybe?
And visual style of 'content' prop in basic Tooltip corresponds to `title` in TooltipLayout, which is not correct - it should be like `description`

Reply:
I'm added new component **ActionTooltip**, that based on Tooltip and sets this layout to his content